index.html

```diff
@@ -18,8 +18,31 @@ <h1>Daniel Eicher</h1>
       </ul>
     </nav>
   </header>
+  <figure>
+    <img src="https://source.unsplash.com/random/600x300" alt="Beispielbild">
+    <figcaption>Dies ist ein Beispielbild</figcaption>
```

Comment:
no code problem - but i believe, you want to create your personal website in English - then i would change this 🇬🇧

Reply:
thx a lot - I will fix it :)